packages/lit/src/components/wizard-color-input/index.ts

```diff
@@ -57,13 +61,28 @@ export class WizardColorInput extends WizardTokenInput {
   };
 
   override render() {
+    const colorValue = this.#token.$value;
+    const hasValidColor = colorValue && Array.isArray(colorValue.components) && colorValue.components.length === 3;
+    const colorString = hasValidColor
+      ? WizardColorInput.supportsCSSColorValues
+        ? this.#token.toCSSColorFunction()
+        : this.#token.toHex()
+      : '#000000';
```

Comment:
nice to have the fallback, but maybe split it up in two declarations so it does not have the nested ternary